packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -114,14 +124,32 @@ export const PageBlock: FunctionComponent<PageBlockProps> = ({
              * so it automatically handles focusing on closest node on margin-clicking
              */
             ".ProseMirror": {
-              padding: `0 ${PAGE_HORIZONTAL_PADDING_FORMULA} 320px`,
+              padding: `0 ${containerPadding[1]} 320px ${containerPadding[0]}`,
```

Comment:
This is… odd? The order is inverted but also – why are we taking a specific style prop like this? 

Also, while we're at it, why is this a global style at all? 

Also does this break the margin clicking stuff?

Reply:
This wasn't a change made in this PR but I guess it's a global style because it's meant to target the page's prosemirror editor. Maybe it could be done through something like `editor.dom.classList.add(...)` ?

I added this here because we are calculating the padding of the page container in `[page-slug].page` but we also need to apply it  here to handle margin-click (at least if we follow the previous implementation, but maybe there's a more elegant solution). The order is inverted because I passed it in the array as `[leftPadding, rightPadding]` but I can invert it back if you think this is confusing.